### ISSUE TITLE
Etcher 1.3.1 detected potential corruption issues during validation 

### ISSUE THREAD

**Issue by Webbanner** [resin-io/etcher#2194]
![etcher_validation](https://user-images.githubusercontent.com/38309548/38659150-ba5d8364-3e5a-11e8-82cb-8b0fdf4e5d00.jpg)
Happened during validating SAMSUNG micro SD card EVO, 16gb

**Comment by Webbanner** [comment]
But successfully validating KINGSTON 8GB micro SD card, class 4

**Comment by lurch** [comment]
Is it possible that your "SAMSUNG UHS class 1" is broken / flaky / faulty?

Sometimes some SD card-readers aren't fully compatible with certain types of SD cards, and it's possible to avoid this problem by using a different SD card-reader.

**Comment by jhermsmeier** [comment]
No, there shouldn't be any issues regarding the SD card classes; any should work.
This is most likely to be an issue with the hardware, or sporadic write errors – i.e. the SD card itself, or the adapter – you can retry it (if you haven't already), and if it consistently fails, then there's something wrong with the hardware.

**Comment by Webbanner** [comment]
Might be the SAMSUNG sd card broken...use same card reader on KINGSTON..it works like charm..

**Comment by rowanbradley** [comment]
I've got the same issue. I am trying to use a 32GB micro-SD card, class 10, marked HC (no other brand marking). I have tried more than once, and I have bought another 32GB card, and it still does it. I really really don't think this is a hardware problem, or a "sporadic write errors" problem. Can anyone suggest how to move forward, i.e. how to overcome this problem, or how to investigate further what is wrong>?

Thanks - Rowan

**Comment by lurch** [comment]
You could use h2testw to check that your SD card is behaving correctly, and not silently corrupting data.

**Comment by tordf** [comment]
Have the same issue. Tried with both a Sandisk Utra 16 GB and a Panasonic 8GB. Both are class 10 HC 1. The Panasonic card is brand new.
